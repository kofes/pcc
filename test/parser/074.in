type
	myArray = array of integer;
	ptr_arr = ^myArray;
begin
end
