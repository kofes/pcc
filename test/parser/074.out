<func table>:
<type table>:
BOOLEAN
CHAR
INTEGER
MYARRAY=ARRAY OF INTEGER
PTR_ARR=^(MYARRAY=ARRAY OF INTEGER)
REAL
<var table>:
PI:C:REAL:3.1415926535897932385
<block statement>:
------------------
